gsad/doc/gsad.8.xml

```diff
@@ -28,26 +28,18 @@
  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
 -->
 
-<manpage name="gsad" section="8" desc="Greenbone Security Assistant for the Open Vulnerability Assessment System (OpenVAS)">
+<manpage name="gsad" section="8" desc="Greenbone Security Assistant Daemon">
 
   <synopsis>
     <cmd>gsad OPTIONS</cmd>
   </synopsis>
 
   <description>
     <p>
-      Together, OpenVAS Scanner (openvassd(8)) and Greenbone Vulnerability Manager
-      (gvmd(8)) provide the core functionality of the Open
-      Vulnerability Assessment System (OpenVAS), with OpenVAS Scanner
-      handling the actual scan process and Greenbone Vulnerability Manager adding
-      various management functionalities to handle and organise scan
-      results.
-    </p>
-
-    <p>
-      The Greenbone Security Assistant Daemon (gsad) is a web
-      application that connects to the Greenbone Vulnerability Manager to provide for a
-      full-featured user interface for vulnerability management.
+      The Greenbone Security Assistant Daemon (gsad) is a stand-alone web server
+      that acts as the layer between the single page application Greenbone
```

Comment:
Could we say front-end application instead of single page application? For me single page application is only a technical detail.